### PR TITLE
CommandLine bug fixes

### PR DESCRIPTION
**Pull request by praskutti**
Form1.cs : Fixed an issue that ends up in the below error when used from CommandLine. Exception Info: System.NullReferenceException: Object reference not set to an instance of an object. 
Program.cs : Added another usage of the commandline. I think file names without file path isnt working. I will have to debug more to understand why File.Exists doesnt work in OpenConfigFile method

### CODE REVIEW

File: src/SQLQueryStress/Program.cs [hunk 1]
```diff
@@ -54,6 +54,7 @@ private static void DisplayHelp<T>(ParserResult<T> result, IEnumerable<Error> er
                     help.AddPreOptionsLine("Check for updates at: https://github.com/ErikEJ/SqlQueryStress");
                     help.AddPreOptionsLine("Sample usage:");
                     help.AddPreOptionsLine("SqlQueryStress -c saved.SqlStress -u -t 32 -d sqldb.perfenv.mycompany.com -r results.csv");
```
Comment (ErikEJ) [inline src/SQLQueryStress/Program.cs:56]: Should this line not be removed?

File: src/SQLQueryStress/Form1.cs [hunk 2]
```diff
@@ -367,7 +367,7 @@ private void OpenConfigFile(string fileName)
             {
                 MessageBox.Show($"{Resources.ErrLoadingSettings}: {ex.Message}");
             }
-
+            //sqlControl1 = new SqlControl();
```
Comment (ErikEJ) [inline src/SQLQueryStress/Form1.cs:370]: Remove the line

File: src/SQLQueryStress/Form1.cs [hunk 3]
```diff
@@ -621,7 +621,7 @@ private void WriteBenchmarkCsvText(TextWriter tw)
         private void Form1_Load(object sender, EventArgs e)
         {
             var elemHost = new System.Windows.Forms.Integration.ElementHost();
-            sqlControl1 = new SqlControl();
+            //sqlControl1 = new SqlControl();
```
Comment (ErikEJ) [inline src/SQLQueryStress/Form1.cs:624]: Remove the line, do not leave a commented code.

**Review by ErikEJ — CHANGES_REQUESTED** [review]

File: src/SQLQueryStress/Program.cs (hunk 1 above)
Comment (praskutti) [reply src/SQLQueryStress/Program.cs:56]: @ErikEJ, I would assume so. But, prior to removing that line, I will do some testing around path names (full file path vs just file name). I will also try to fix the code to accept both versions.

File: src/SQLQueryStress/Form1.cs (hunk 2 above)
Comment (praskutti) [reply src/SQLQueryStress/Form1.cs:370]: @ErikEJ , I will remove that line.

File: src/SQLQueryStress/Form1.cs (hunk 3 above)
Comment (praskutti) [reply src/SQLQueryStress/Form1.cs:624]: @ErikEJ , I will remove that line.

**Comment by ErikEJ** [comment]
Please limit this PR to bug fixing, and create a new one for new features.

**Comment by ErikEJ** [comment]
Are you still working on this? If not, I will close it

**Pull request closed without merging**